src/job-feed/JobDetailsSection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  Card, CardContent, Typography,
+} from '@material-ui/core';
+
+function JobDetailsField({ title, data, component }) {
```

Comment:
It is a good idea to split it into smaller components but in this case there's not much point, considering the comments below, you can just run `data.map` within `CardContent` and only have  a single `JobDetailsSection` component which would be sufficiently small